sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TracerProvider.java

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Need unit tests for this class.

Reply:
Can you please give the specifics, what all scenario would be good for unit testing.

Reply:
@samvaity - good idea, can you please suggest some ideas on how other SDKs are testing this and the coverage of `TracerProvider`? 
I am tracking the issue here : https://github.com/Azure/azure-sdk-for-java/issues/13032
Can you please provide the context on the issue and we will take care of it. 
@simplynaveen20 - please take this up on a follow up PR.

Reply:
Some existing examples of unit testing this class could be found here https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/TracerProviderTest.java

It may need to be updated (or not) to fit Cosmos scenarios.

Reply:
as mentioned by Kushagra , tracking in #13032